openlibrary/solr/data_provider.py

```diff
@@ -143,13 +143,17 @@ async def _get_lite_metadata(ocaids: list[str], _recur_depth=0, _max_recur_depth
                 r = await client.get(
                     "https://archive.org/advancedsearch.php",
                     timeout=30,  # The default is silly short
+                    headers={
+                        'x-application-id': 'ol-solrbuilder',
```

Comment:
Switching between `solrbuilder` and `solrupdater` would be annoying. Let's just use `ol-solr` for now unless we notice abnormalities in traffic.

```suggestion
                        'x-application-id': 'ol-solr',
```